geospatial/dted.ksy

```diff
@@ -0,0 +1,596 @@
+          for values less than 100; H is the hemisphere of the data (DDDMMSSH).
+      - id:  se_corner_of_data
+        type: coordinate_pair_ddmmssh
+        doc: |
+          Latitude of SE corner of data, bounding rectangle—leading zero for |
+          values less than 10; H is the hemisphere of the data.
+          Longitude of SE corner of data, bounding rectangle— leading zeroes |
+          for values less than 100; H is the hemisphere of the data.
+      - id:  clockwise_orientation_angle
+        type: angle_dddmmss_sh
+        doc: |
+          Clockwise orientation angle of data with respect to true North. |
+          (Will usually be all zeros.)
+      - id:  latitude_interval
+        type: ascii_int
+        doc: |
+          Latitude interval in tenths of seconds between rows of elevation |
+          values (SSSS).
+      - id:  longitude_interval
+        type: ascii_int
+        doc: |
+          Longitude interval in tenths of seconds between columns of elevation |
+          values (SSSS).
+      - id:  number_of_latitude_lines
+        type: ascii_int
+        doc: |
+          Number of Latitude lines. For magnetic tape, this is the actual |
+          count of the number of latitude points (rows that contain data). |
+          For CD-ROM, this is the count of the number of latitude points in |
+          a full one-degree cell. (e.g. 1201 for DTED1, 3601 for DTED2) |
+          (0000-9999).
+      - id:  number_of_longitude_lines
+        type: ascii_int
+        doc: |
+          Number of Longitude lines. For magnetic tape, this is the actual |
+          count of the number of longitude points (columns that contain data). |
+          For CD-ROM, this is the count of the number of longitude points in a |
+          full one-degree cell. The count is based on the level of DTED and |
+          the latitude zone of the cell. (See Table II and III) (0000-9999). 
+      - id:  partial_cell_indicator
+        type: str
+        size: 2
+        doc: |
+          Partial Cell Indicator 
+          00 = Complete 1° cell 
+          01-99 = % of data coverage.
+      - id:  reserved_for_nima_use_only
+        type: str
+        size: 101
+        doc: |
+          Reserved for NIMA use only. (Free text or Blank filled.)
+      - id:  reserved_for_producing_nation_use_only
+        type: str
+        size: 100
+        doc: |
+          Reserved for producing nation use only. (Free text or blank filled.)
+      - id:  reserved_for_free_text_comments
+        type: str
+        size: 155
+        doc: |
+          Reserved for free text comments. (Free text or Blank filled.)
+  acc_header:
+    doc: |
+      Accuracy Description (ACC) Record. The ACC is used for magnetic tape and |
+      CD-ROM. Fixed Length = 2700 ASCII Characters |
+      Note: *If Product has subregional accuracies, the overall accuracy of |
+      the product will be the worst accuracy.
+    seq:
+      - id: recognition_sentinel
+        contents: ACC
```

Comment:
I wonder if it is a marker of block type, so if a generic block type should be created and specific block type is selected with
```yaml
type:
    switch-on: recognition_sentinel
    cases:
      '"ACC"': acc_header
```

Reply:
An interesting suggestion.  It certainly could be implemented that way.  The spec doesn't seem to allow for that as the spec reads as follow (page 13)
```
						Record starts at byte
a. User Header Label (UHL: 80 bytes)			1
b. Data Set Identification Record (DSI: 648 bytes)	81
c. Accuracy Record (ACC: 2700 bytes)			729
d. Data Records (Each data record is 2414 bytes)*	3429
							5843
							8257
							etc
```